src/components/radioGroup/__tests__/index.spec.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import {render, fireEvent} from '@testing-library/react-native';
+import RadioGroup from '../index';
+import RadioButton from '../../radioButton/index';
+
+const testIDs = {
+  radioGroup: 'RadioGroup',
+  radioButtonUp: 'radioButtonUp',
+  radioButtonDown: 'radioButtonDown',
+  radioButtonLeft: 'radioButtonLeft',
+  radioButtonRight: 'radioButtonRight'
+};
+const onValueChangeMock = jest.fn();
+
+const RadioGroupTestComponent = props => {
+  const {shouldDisable, ...others} = props;
+  return (
+    <RadioGroup {...others} onValueChange={onValueChangeMock} testID={testIDs.radioGroup}>
+      <RadioButton value={'up'} label={'Up'} disabled={shouldDisable} testID={testIDs.radioButtonUp}/>
+      <RadioButton value={'down'} label={'Down'} testID={testIDs.radioButtonDown}/>
+      <RadioButton value={'left'} label={'Left'} testID={testIDs.radioButtonLeft}/>
+      <RadioButton value={'right'} label={'Right'} testID={testIDs.radioButtonRight}/>
+    </RadioGroup>
+  );
+};
+
+describe('RadioGroup renderer test', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('Press on radio button', async () => {
+    const props = {onValueChangeMock};
+    const element = <RadioGroupTestComponent {...props}/>;
+
+    const {getByTestId} = render(element);
+
+    const radioGroup = getByTestId(testIDs.radioGroup);
+    expect(radioGroup).toBeTruthy();
```

Comment:
This line is redundant since `getByTestId` already checks if `testIDs.radioGroup` is exist.